packages/static-wado-scp/lib/program/index.js

```diff
@@ -2,8 +2,16 @@ const staticWadoUtil = require("@ohif/static-wado-util");
 const { DcmjsDimseScp, Server } = require("..");
 const packageJson = require("../../package.json");
 
+const aeConfig = staticWadoUtil.aeConfig;
+
 function main() {
-  const port = this.dicomWebScpConfig.scpPort || 11112;
+  const { queryAe } = this.dicomWebScpConfig;
```

Comment:
It isn't queryAe that is used for this AE configuraiton, but choose another name.  The queryAe the ae name of the back end that is being proxied to.  How about staticWadoAe, defined by default in staticWadoConfig, and inherited into both teh scp and webserver.